### PR TITLE
Split the dashboards into multiple files

### PR DESCRIPTION
**Pull request by vglafirov**
* simplifies the import/export of the dashboards for kube-prometheus

### CODE REVIEW

**Comment by gianrubio** [comment]
@vglafirov if you want to do this changes you should edit the [sync_kube_prometheus](https://github.com/coreos/prometheus-operator/blob/master/helm/hack/sync_kube_prometheus.py) script

**Comment by vglafirov** [comment]
@gianrubio  Please review it again

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (gianrubio) [inline .vscode/settings.json]: please remove this file

**Comment by vglafirov** [comment]
@gianrubio Done.

**Comment by vglafirov** [comment]
Build succeeded, but check still in pending for some reason :(

**Comment by gianrubio** [comment]
CI is green, probably travis has some issues communicate to GitHub. Please give more time to validate the changes, thanks!

**Comment by vglafirov** [comment]
it's been like this for three days already though ;(

**Comment by vglafirov** [comment]
Added myself to the list of maintainers in order to trigger the build again

File: vendor/github.com/prometheus/client_model/.project [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (gianrubio) [inline vendor/github.com/prometheus/client_model/.project]: please remove all this files

**Comment by bgagnon** [comment]
What about [/contrib/kube-prometheus/manifests/grafana/grafana-dashboard-definitions.yaml](https://github.com/coreos/prometheus-operator/blob/master/contrib/kube-prometheus/manifests/grafana/grafana-dashboard-definitions.yaml)? Any plans to split that one as well?

**Comment by brancz** [comment]
We're currently reworking the Grafana stack of kube-prometheus, at this point it is unclear what the final result will look like, but I believe that it will stay a single ConfigMap (bin-packed if necessary). (https://github.com/brancz/kubernetes-grafana)

**Comment by vglafirov** [comment]
@bgagnon This is not helm template, but pure k8s manifests. There no reason or even possibility to do the same trick there, unfortunately.

**Comment by vglafirov** [comment]
@brancz Shall I close this PR and start contributing to new repo? We are going to use this in production, so I motivated to get this more reliable and maintainable.

**Comment by brancz** [comment]
I'm personally not convinced that helm is the solution we should be using to deploy and manage our applications on top of Kubernetes, but the rest of the tooling is also underdeveloped, so we (mostly @gianrubio, who's doing a fantastic job) are maintaining the the helm charts as the community wants charts. The charts will continue to exist, but we'll be working on another solution to manage this stack in addition to helm.

**Comment by vglafirov** [comment]
@brancz Let's may be merge this PR then :) I will continue contributing to these charts, as helm is our main deployment tool.

**Comment by brancz** [comment]
@gianrubio maintains the charts, he makes the call :slightly_smiling_face:

**Comment by gianrubio** [comment]
I bumped the charts version, on green I'll merge it. Thanks @vglafirov

**Comment by gianrubio** [comment]
@vglafirov yesterday I introduce a change in the travis-ci script to solve the failure that we got now in the CI, could you rebase the code again?

**Comment by vglafirov** [comment]
@gianrubio All green now

**Comment by vglafirov** [comment]
@deyceg I see. I will fix that. Thanks

**Pull request merged**